apps/native-tests/ios/NativeTests.xcodeproj/project.pbxproj

```diff
@@ -401,6 +402,10 @@
 				"EXCLUDED_ARCHS[sdk=iphonesimulator*]" = "";
 				GCC_C_LANGUAGE_STANDARD = gnu99;
 				GCC_NO_COMMON_BLOCKS = YES;
+				GCC_PREPROCESSOR_DEFINITIONS = (
+					"$(inherited)",
+					_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION,
```

Comment:
i would recommend not to commit this xcode 15 workaround.
soon after 0.73 upgrade, we don't need this anymore.